pdns/mastercommunicator.cc

```diff
@@ -93,7 +95,7 @@ void CommunicatorClass::queueNotifyDomain(const DNSName &domain, UeberBackend *B
     }
   }
 
-  if (!hasQueuedItem)
+  if (!hasQueuedItem && d_onlyNotify.size())
```

Comment:
Your comment is wrong. I have PDNS with slave zones only and this slave zones should not send NOTIFYs to the NS-RRs. Some if my slave zones have further slave zones, thus I have slave-renotify enabled.
So this warning pops up every time a zone gets updated although everthing is correct.
It does not make sense to print out this warning if NS-RR NOTIFYs where explicitely disabled.

In my opinion this Warning was wrong from the beginning. There should not be a warning if a zone does not have any also-notifys (that's fine). IMO the warning makes only sense if the only-notify feature blocked all NS-RR NOTIFYs.